### ISSUE TITLE
How to monitor incoming transactions?

### ISSUE THREAD

**Issue by akegaviar** [akegaviar/ton-foo#314]
This might be a very basic question, but if I want to handle incoming transactions to my smart contract and react to them in some way, how should I monitor them?

This question was imported from Telegram Chat: [https://t.me/tondev/119038](https://t.me/tondev/119038)

## Answer

You can take a look at the resources available at [https://github.com/gobicycle/bicycle](https://github.com/gobicycle/bicycle) and [https://docs.ton.org/develop/dapps/asset-processing/](https://docs.ton.org/develop/dapps/asset-processing/) to learn more about this.

Basically, you can track incoming transactions to a specific address.

---

[Original](https://answers.ton.org/question/1561419580209893376/how-to-monitor-incoming-transactions)
